bitsail-common/pom.xml

```diff
@@ -88,11 +88,6 @@
             <artifactId>commons-csv</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>joda-time</groupId>
```

Comment:
Curious is this change related to this PR? I assume it's a redundant dependency that we can still build the project without it?

Reply:
Yeah it was a duplicate. Looks like someone else fixed it already so no change on this one.